sklearn/cross_validation.py

```diff
@@ -60,6 +61,11 @@
            'permutation_test_score',
            'train_test_split']
 
+@deprecated("FitFailedWarning has been moved into the sklearn.exceptions"
+            " module. It will not be available here from version 0.19")
+class FitFailedWarning(_FitFailedWarning):
```

Comment:
Thanks for the catch!

And the thing is it was available publicly in `sklearn.cross_validation.FitFailedWarning` and hence cannot be removed without deprecation.

I'll refactor it according to your suggestion (`MyDeprecatedFitFailedWarning = deprecated('Bla')(FitFailedWarning)`)